### PR TITLE
[MODIFY] Filter DO

### PR DESCRIPTION
**Pull request by watcharaporn-git**

### CODE REVIEW

File: custom-addons/esb/packing_list/models/packing_list.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ class ESBPackingList(models.Model):
     picking_ids = fields.Many2many(
         comodel_name="stock.picking",
         string="Packing List",
-        domain="[('partner_id', '=', partner_id)]",
+        domain="[('partner_id','=',partner_id),('location_dest_id', '=', 5)] ",
```
Comment (ps-tubtim) [inline custom-addons/esb/packing_list/models/packing_list.py:123]: Should you filter by `picking_type_id`? I think it better than filter by `location_dest_id`.

**Review by ps-tubtim — APPROVED** [review]

File: custom-addons/esb/sale/models/sale.py [hunk 2]
```diff
@@ -24,6 +23,10 @@ class SaleOrder(models.Model):
         string="Bank Account",
         domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]",
     )
+    type_id = fields.Many2one(
+        comodel_name="sale.order.type",
+        domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]",
+    )
```
Comment (ps-tubtim) [inline custom-addons/esb/sale/models/sale.py:29]: If you create a pull request at OCA/sale-workflow to fix it, this code is unnecessary. You should edit `repos.yaml` instead.

File: custom-addons/esb/sale/models/sale.py [hunk 3]
```diff
@@ -64,6 +77,8 @@ class SaleOrderLine(models.Model):
         currency_field="currency_id",
         help="Subtotal not including discount",
     )
+    # company_id = fields.Char('company_id'),
+    # company_ids = fields.Char('sale_order_type.company_id')
```
Comment (ps-tubtim) [inline custom-addons/esb/sale/models/sale.py:81]: I think this code unnecessary.

**Comment by kittiu** [comment]
Moved to OCA

**Pull request closed without merging**